MQTTnet.Core/Client/MqttClientQueuedOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿
+
+namespace MQTTnet.Core.Client
+{
+    public class MqttClientQueuedOptions: MqttClientTcpOptions
+    {
+        public bool UsePersistence { get; set; }
```

Comment:
This is not required. The property with the "Storage" can be empty. Then this feature should be disabled at all. With this additional property you can set this to true even without a Storage implementation.

Reply:
Right

Reply:
Done